samples/integration/cmake-vcpkg/main.cpp

```diff
@@ -0,0 +1,41 @@
+#include <azure/identity.hpp>
+#include <azure/keyvault/secrets.hpp>
+
+#include <iostream>
+
+using namespace Azure::Security::KeyVault::Secrets;
+
+int main()
+{
+  std::cout << "Starting Program!" << std::endl;
+
+  try
+  {
+    // Set Key Vault URL string
+    auto const keyVaultUrl = std::getenv("AZURE_KEYVAULT_URL");
+
+    // Create Default Azure Credential to Authenticate. I will pick up on our AzureCLI login
```

Comment:
You created the AzureCLI credential. The comment mentions Default Azure Credential. Which one did you mean to use?